### ISSUE TITLE
Feature request: handle subscription offers

### ISSUE THREAD

**Issue by orestis-z** [cozycodegh/cordova-plugin-inapppurchases#3]
Google play has the option to add subscription offers (e.g. 7 day free trial). I didn't find a way to handle this in the implementation of this package.
Are there any plans to add it?

**Comment by cozycodegh** [comment]
Are you looking to get a list of details on all of the available subscriptions to get the price after the trial? I can add that to list of updates to make next.

With this response, it should subscribe you to the plan including the free trial.

**Comment by orestis-z** [comment]
Yes, that would be perfect. It would be sufficient to get the details of the price after the trial. Thanks a lot!

**Comment by cozycodegh** [comment]
Thanks for the suggestion, this is now in the latest version of the plugin (it will return introductoryPrice and introductoryOriginalPrice, just like on iOS)

**Comment by orestis-z** [comment]
That's great! Thanks a lot!
Could we also add `introductoryOriginalPriceAsDecimal`? Currently, in order to get this information as a floating point number I'd need to deduce it from `introductoryOriginalPrice` (e.g. `4,79 €`) which is not ideal since doesn't have a fixed format

**Comment by cozycodegh** [comment]
Sounds good! That’s now in the newest version too
